src/overlays/actors/ovl_Bg_Mizu_Bwall/z_bg_mizu_bwall.c

```diff
@@ -217,7 +220,15 @@ void BgMizuBwall_Init(Actor* thisx, PlayState* play) {
                 }
             }
             break;
-        case MIZUBWALL_RUTO_ROOM:
+        }
+        case MIZUBWALL_RUTO_ROOM: {
```

Comment:
```c
} break;

case ...
```

Reply:
See PR description, break outside of the block didn't work in a couple places in this function so I changed them all for consistency

Reply:
Oh interesting
Sorry didn't pay attention to the description